readme.md

```diff
@@ -359,7 +361,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [bpftrace](https://bpftrace.org/) - A tool for tracing with its own high-level tracing language. It is flexible enough to be envisioned as a Linux replacement for DTrace and SystemTap.
   - [bpftrace Cheat Sheet](https://www.brendangregg.com/BPF/bpftrace-cheat-sheet.html) - Summary and cheat sheet for programming in bpftrace. Contains information about syntax, probe types, variables and functions.
 - [kubectl trace](https://github.com/iovisor/kubectl-trace) - A kubectl plug-in for executing bpftrace programs in a Kubernetes cluster.
-- [inspektor-gadget](https://github.com/inspektor-gadget/inspektor-gadget) - A collection of eBPF-based tools to debug and inspect Kubernetes resources and applications.
+- [inspektor-gadget](https://github.com/inspektor-gadget/inspektor-gadget) - A collection tools and framework for data collection and system inspection on Kubernetes clusters and Linux hosts using eBPF.
```

Comment:
I think I had not seen the official website (https://inspektor-gadget.io) for the project before. I'm pretty sure it didn't exist when this entry in the list was created. Would it make sense to update the link to point to the website? The GitHub repo is accessible from there.

Reply:
> Would it make sense to update the link to point to the website? The GitHub repo is accessible from there.

Good catch! Yes, using website makes more sense!